ruff_lsp/server.py

```diff
@@ -475,14 +830,17 @@ async def code_action(params: CodeActionParams) -> list[CodeAction] | None:
                 and len(params.context.only) == 1
                 and kind in params.context.only
             ):
-                edits = await _fix_document_impl(document, only="I001")
-                if edits:
+                workspace_edit = await _fix_document_impl(
+                    Document.from_cell_or_text_uri(params.text_document.uri),
+                    only="I001",
+                )
+                if workspace_edit:
```

Comment:
Code actions now work at cell level for a Notebook. This is consistent with everything else in the VS Code ecosystem and a user can use builtin commands (`Ruff:`) to run an action on the entire notebook.